static/app/views/traces/content.tsx

```diff
@@ -796,13 +841,13 @@ const StyledPanel = styled(Panel)`
 const TracePanelContent = styled('div')`
   width: 100%;
   display: grid;
-  grid-template-columns: repeat(1, min-content) auto repeat(2, min-content) 85px 112px 66px;
+  grid-template-columns: 116px auto repeat(2, min-content) 85px 112px 66px;
 `;
 
 const SpanPanelContent = styled('div')`
   width: 100%;
   display: grid;
-  grid-template-columns: repeat(1, min-content) auto repeat(1, min-content) 160px 85px;
+  grid-template-columns: 100px auto repeat(1, min-content) 160px 85px;
```

Comment:
Why? doesn't this cover ids?

Reply:
we dont use a monospace font for the id so depending on the id, the trace/span id could end up with different widths using min-content

Reply:
Ah now you say it I did notice that